man/tpm2_pcrallocate.1.md

```diff
@@ -0,0 +1,57 @@
+% tpm2_pcrallocate(1) tpm2-tools | General Commands Manual
+%
+% MARCH 2019
+
+# NAME
+
+**tpm2_pcrallocate**(1) - Change the allocated PCRs of a TPM
+
+# SYNOPSIS
+
+**tpm2_pcrallocate** [*OPTIONS*] [*ALLOCATION*]
+
+# DESCRIPTION
+
+**tpm2_pcrallocate**(1) - allows the user to specify an allocation for the TPM.
+If no *ALLOCATION* is given, then SHA1 and SHA256 banks with PCRs 0 - 23 are
+allocated.
+
+# ALLOCATION
+
+A list of banks and selected pcrs. To allocate a new bank a hash algorithm followed
+by a colon is presented, `sha1:` followed by the keyword `all` or a space separated
+list of pcrs; e.g. `sha1: all` or `sha256: 0 1 5 15 16`. Multiple of these bank
+definitions can be presented sequentially; e.g. `sha1: all sha256: all`. If no
```

Comment:
Worth pointing out that if user specify the bank multiple time, e.g
`sha256: all .... sha256: 0 1 5 16`
the behaviour should be that the LAST one will be applied

Reply:
Wrote a sentence to disallow it. Will add program code for this as well.

Reply:
Using pcr_parse_selections() and common/pcrs.md now